sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/YAMLToInternalMappers.java

```diff
@@ -122,15 +122,15 @@ private static DefaultServiceSpec convertServiceSpec(
     private static void verifyDistinctDiscoveryPrefixes(Collection<RawPod> rawPods) {
         Map<String, Long> dnsPrefixCounts = rawPods.stream()
                 .flatMap(p -> p.getTasks().values().stream()
-                        .map(t -> t.getDiscovery())
-                        .filter(d -> d != null)
-                        .map(d -> d.getPrefix())
-                        .filter(prefix -> prefix != null)
+                        .map(RawTask::getDiscovery)
```

Comment:
Although this is a good change, keep in mind that this would trigger the entire CI and may be better suited to a follow-up PR. I'm happy to leave it in though.

Reply:
Assuming we control the trigger vs non-trigger of the CI on teamcity VCS triggers based on file extensions, since this CI already has some diff in *.py files, it would already trigger the entire CI right? Or am I missing something?